core/src/main/java/org/jruby/lexer/StrptimeLexer.java

```diff
@@ -10,7 +17,7 @@
 /**
  * This class is a scanner generated by 
  * <a href="http://www.jflex.de/">JFlex</a> 1.4
- * on 5/6/17 4:40 AM from the specification file
+ * on 6/16/17 3:03 PM from the specification file
  * <tt>core/src/main/java/org/jruby/lexer/StrptimeLexer.flex</tt>
  */
 public class StrptimeLexer {
```

Comment:
I re-generated .java file. If we don't need the update, please let me know. I will remove it.